docs/public-api/public-api.v2.yaml

```diff
@@ -180,6 +180,11 @@ paths:
                 properties:
                   run:
                     $ref: '#/components/schemas/Run'
+                  differences:
+                    type: array
+                    description: Only present if the `diff` parameter is set to `true` and there is an unambiguous previous commit with a run
```

Comment:
I assume the 'diff' parameter is the 'diff_prev' one? In which view do we need a run without differences?

Reply:
I don't know any (we only hit that endpoint in the detail page AFAICT), but it might be nice for casual scripts.

Reply:
Ah, I see. I couldn't think of any, so I was wondering. But you are right, that might come in useful.